build.gradle.kts

```diff
@@ -38,25 +46,29 @@ dependencies {
   runtimeOnly("org.postgresql:postgresql:42.5.1")
   runtimeOnly("org.flywaydb:flyway-core")
 
-  testImplementation("org.springframework.security:spring-security-test")
-  testImplementation("io.jsonwebtoken:jjwt:0.9.1")
+  annotationProcessor("org.springframework.boot:spring-boot-configuration-processor")
 
-  testImplementation("org.mockito:mockito-inline:4.11.0")
+  testImplementation("io.swagger.parser.v3:swagger-parser:2.1.11")
+  testImplementation("com.github.tomakehurst:wiremock-jre8-standalone:2.35.0")
+  testImplementation("org.mockito:mockito-inline:5.1.1")
+  testImplementation("org.testcontainers:localstack:1.17.6")
   testImplementation("org.awaitility:awaitility-kotlin:4.2.0")
-
-  testImplementation("com.github.tomakehurst:wiremock-standalone:2.27.2")
+  testImplementation("javax.xml.bind:jaxb-api:2.3.1")
+  testImplementation("org.springframework.boot:spring-boot-starter-test")
+  testImplementation("io.jsonwebtoken:jjwt:0.9.1")
+  testImplementation("org.springframework.security:spring-security-test")
   testImplementation("org.testcontainers:postgresql:1.17.6")
   testImplementation("org.testcontainers:localstack:1.17.6")
 }
 
 java {
-  toolchain.languageVersion.set(JavaLanguageVersion.of(17))
+  toolchain.languageVersion.set(JavaLanguageVersion.of(18))
```

Comment:
this can go to 19

Reply:
I thought you might say that will upgrade